src/crypto/transform.py

```diff
@@ -209,41 +180,33 @@ def trading_volume_fact_rows(self, date_key: date, crypto_data: dict) -> list:
         crypto_data = crypto_data["data"]
 
         # name dimension
-        name_dim_rows = transform.name_dim_rows(crypto_data=crypto_data)
-        if name_dim_rows:
+        if (name_dim_rows := transform.name_dim_rows(crypto_data=crypto_data)):
```

Comment:
sorry ;)

it's ok but maybe having the extra line is more readable, see some of the feedback here: https://twitter.com/bbelderbos/status/1687044213790130176

Reply:
of course for now leave it like this, but just a consideration, I think we can live long and happily without using Walrus ;)